src/decorators.ts

```diff
@@ -1,4 +1,3 @@
-import 'reflect-metadata';
```

Comment:
`reflect-metadata` is needed on this file

Reply:
NO

Reply:
Yes? If you take it out Typescript throws errors because it cannot figure out where references to Reflect come from.